manifests/lib/kube.libsonnet

```diff
@@ -384,9 +384,6 @@
 
       updateStrategy: {
         type: "RollingUpdate",
-        rollingUpdate: {
-          partition: 0,
-        },
```

Comment:
Fixes the rollout issue for statefulsets. From https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#partitions it appears that it does not make sense to have it defined by default.

Reply:
/cc @anguslees

Reply:
Nice find!

Oddly, the default for that value is meant to be 0, but it's actually a `*int32`, so `nil` is different to `0`.  So much for the docs :stuck_out_tongue: